1-js/03-code-quality/03-comments/article.md

```diff
@@ -145,6 +145,6 @@ Any subtle features of the code? Where they are used?
 
 ## Summary
 
-One of signs of a good developer is his comments. Good comments allow to maintain the code well, return to it after a long delay and use features more effectively.
+One sign of a good developer is his comments. Good comments allowus  to maintain the code well, return to it after a long delay and use features more effectively.
```

Comment:
@iliakan Sorry! this is my mistake. I will fix it, and I will be more careful next time :)